src/main/java/core/basesyntax/Drawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawing {
```

Comment:
invalid interface name
```suggestion
public interface Drawable {
```